build.sbt

```diff
@@ -23,8 +23,35 @@ lazy val akkaProjectionCore = Project(
   ).settings(Dependencies.core)
 
 
+// provides Runner and OffsetStore for transactional projections
+// lazy val akkaProjectionJdbc = Project(
+//   id = "akka-projection-jdbc",
+//   base = file("akka-projection-jdbc")
+// ).settings(Dependencies.jdbc)
+
+// provides Sources backed by Alpakka 
+// and Runners that commits on Topic (at-least-once, at-most-once)
+lazy val akkaProjectionAlpakkaKafka = Project(
+  id = "akka-projection-alpakka-kafka",
+  base = file("akka-projection-alpakka-kafka")
+).settings(Dependencies.alpakkaKafka)
+  .settings(libraryDependencies ++= Seq(Dependencies.Test.scalaTest))
+  .dependsOn(akkaProjectionCore, akkaProjectionMocks)
+
+
+lazy val akkaProjectionMocks = Project(
+  id = "akka-projection-mocks",
+  base = file("akka-projection-mocks")
+).settings(libraryDependencies ++= Seq(Dependencies.Test.scalaTest))
+  .dependsOn(akkaProjectionCore)
```

Comment:
In #7, this was part of the PoC module. Here I start to move it to a mocks module that we can use whenever we want to write tests for one or other integration point. 

For instances, when writing tests for Kafka, I want to have a fake, in-memory, DB to save the projection and the offset store.